docs/install.md

```diff
@@ -1,93 +1,58 @@
 <!--
 ---
-linkTitle: "Installation"
+linkTitle: "Setting Up Tekton Triggers"
 weight: 2
 ---
 -->
-# Installing Tekton Triggers
+# Setting Up Tekton Triggers
 
-Use this page to add the component to an existing Kubernetes cluster.
+Follow the steps below to set up an official release of Tekton Triggers on your Kubernetes cluster. 
+If you want to test Triggers from `HEAD`, see the
+[Tekton Developer Guide](https://github.com/tektoncd/triggers/blob/master/DEVELOPMENT.md#install-triggers).
 
-- [Pre-requisites](#pre-requisites)
-- [Versions](#versions)
-- [Installing Tekton Triggers](#installing-tekton-triggers-1)
+## Prerequisites
 
-## Pre-requisites
+You must satisfy the following prerequisties to set up Tekton Triggers:
 
-1.  A Kubernetes cluster version 1.18 or later
+* Your cluster must be running Kubernetes version 1.18 or above.
 
-    If you don't already have a cluster, you can create one for testing with
-    `kind`.  [Install
-    `kind`](https://kind.sigs.k8s.io/docs/user/quick-start/#installation) and
-    create a cluster by running [`kind create
-    cluster`](https://kind.sigs.k8s.io/docs/user/quick-start/#creating-a-cluster).
-    This will create a cluster running locally, with RBAC enabled.
+* You have installed Tekton Pipelines on your cluster. For instructions, see [Installing Tekton Pipelines](https://github.com/tektoncd/pipeline/blob/master/docs/install.md).
 
-1.  Grant current user `cluster-admin` privileges.
+* You have granted the `cluster-admin` privilege to the user with which you installed Tekton Pipelines. For instructions, see
+  [Role-based access control](https://cloud.google.com/kubernetes-engine/docs/how-to/role-based-access-control#prerequisites_for_using_role-based_access_control).
 
-    _See
-    [Role-based access control](https://cloud.google.com/kubernetes-engine/docs/how-to/role-based-access-control#prerequisites_for_using_role-based_access_control)
-    for more information._
+## Installing Tekton Triggers on Your Cluster
 
-1.  Install Tekton Pipelines. You can install the latest version using the
-    command below or follow the
-    [pipeline installation guide](https://github.com/tektoncd/pipeline/blob/master/docs/install.md):
+1. Open a terminal on your cluster and log in as the user with which you installed Tekton Pipelines.
```

Comment:
This seems a bit misleading to me. You don't open a terminal on your cluster technically. You mainly need a `kubeconfig` which is pointing at the cluster with the correct role.

Reply:
Fair! Reworded.